src/routes/artist.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+
+const artistController = require('../controllers/artist.js');
+const albumController = require('../controllers/albums.js');
+const router = express.Router();
+
+router.get('/', artistController.retrieveArtists)

```

Comment:
These all can be grouped together, the extra lines can be deleted between the routes.